nodejs/week1/meals/src/backend/routes/largemealsroute.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const router = express.Router();
+
+const fs = require('fs');
+
+
+
+const meals = JSON.parse(fs.readFileSync(__dirname + '/../data/meals.json'));
+const reviews = JSON.parse(fs.readFileSync(__dirname + '/../data/reviews.json'));
+
+
+router.get('/', (request, response) =>{
+  const largeMeals = meals.filter(meal => {
+    return meal.maxNumberOfGuests > 8; //parseInt(meal.maxNumberOfGuests > 8);
+  })
+  largeMeals.forEach(meal => {
+    const largemeal = reviews.filter(review => {
+      return review.mealId === meal.id
+    })
+  })
+
```

Comment:
on the route /largemeals, array of meals does not have a review.
There is an example how to do it in cheapmeals, using .map function